tests/phpunit/integration/Modules/AdsTest.php

```diff
@@ -92,6 +92,30 @@ public function test_is_connected__when_ads_conversion_id_is_set() {
 		$this->assertTrue( $this->ads->is_connected() );
 	}
 
+	public function test_is_connected__when_pax_conversion_id_is_set() {
+		$this->assertFalse( $this->ads->is_connected() );
+
+		self::enable_feature( 'adsPax' );
+
+		$this->ads->get_settings()->set(
+			array( 'paxConversionID' => 'AW-123456789' )
+		);
+
+		$this->assertTrue( $this->ads->is_connected() );
+	}
+
+	public function test_is_connected__when_ext_customer_id_is_set() {
+		$this->assertFalse( $this->ads->is_connected() );
+
+		self::enable_feature( 'adsPax' );
+
+		$this->ads->get_settings()->set(
+			array( 'extCustomerID' => '123456789' )
+		);
+
+		$this->assertTrue( $this->ads->is_connected() );
+	}
```

Comment:
We should add a test that ensures the feature flag is working too, and that `paxConversionID` and `extCustomerID ` are ignored when the `adsPax` feature is disabled. Because users will be able to toggle the feature with a plugin, we should ensure that enabling the PAX ads feature, adding their ID via PAX, then disabling the PAX ad feature, won't return the module as connected if it's missing the manually-entered ID.

Reply:
Good point, thanks